pinc/user_project_info.inc

```diff
@@ -247,10 +253,10 @@ function notify_project_event_subscribers($project, $event, $extras = [])
 // -----------------------------------------------------------------------------
 
 function get_n_users_subscribed_to_events_for_project($projectid)
-// Return an array: keys are the same as $subscribable_project_events,
+// Return an array: keys are the same as array returned by get_subscribable_project_events(),
 // each value is the number of subscriptions to that event for the given project.
 {
-    global $subscribable_project_events;
+    $subscribable_project_events = get_subscribable_project_events();
     $items = [];
     foreach ($subscribable_project_events as $event => $label) {
```

Comment:
What you have works great. Note that you could have also removed the variable entirely with:
```php
foreach (get_subscribable_project_events() as $event => $label) {
```